GardenApp/app/src/main/java/com/cs1530_group1/gardenapp/ViewSpeciesInfoActivity.java

```diff
@@ -0,0 +1,151 @@
+ * 
+ * @author Charles Smith <cas275@pitt.edu>
+ */
+
+@SuppressWarnings("deprecation")
+public class ViewSpeciesInfoActivity extends ActionBarActivity {
+
+    private static final String LOG_TAG = "Species Info Activity";
+
+    private String speciesName;
+
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_view_species_info);
+
+        //get the garden from App class
+        Garden garden = ((App) getApplication()).getGarden();
+
+        //get the speciesName passed from the last activity
+        speciesName = getIntent().getStringExtra(Intent.EXTRA_TEXT);
+
+
+        if(speciesName == null||speciesName.isEmpty()) {
+            Toast.makeText(getApplicationContext(),"Error loading information", Toast.LENGTH_SHORT).show();
+            Log.e(LOG_TAG, "received a null or empty string, cannot display relevant information");
+            return;
+        }
+
+        //get the name TextView and append the species name
+        TextView name = (TextView) findViewById(R.id.species_name);
+        String nameString = "<b>Name:</b><br />"+speciesName;
+        name.setText(Html.fromHtml(nameString)); //Html.fromHtml is needed to set bold strings
+
+        //get the description TextView and append info
+        TextView descrpit = (TextView)findViewById(R.id.species_descript);
+        String descriptString = "<b>Description:</b><br />" + garden.getDescription(speciesName);
+        descrpit.setText(Html.fromHtml(descriptString));
+
+        //set type TextView
+        TextView type = (TextView)findViewById(R.id.species_type);
+        String typeString = "<b>Type:</b><br />" + garden.getSpeciesType(speciesName);
+        type.setText(Html.fromHtml(typeString));
+
+        //set sun TextView
+        TextView sun = (TextView)findViewById(R.id.species_sun);
+        String sunString = "<b>Sun Level:</b><br />" + garden.getSunLevel(speciesName);
+        sun.setText(Html.fromHtml(sunString));
+
+        //set plant date TextView
+        TextView plantDate = (TextView)findViewById(R.id.species_plant_date);
+        String plantDateString = "<b>Plant Date:</b><br />"+garden.getPlantDate(speciesName);
+        plantDate.setText(Html.fromHtml(plantDateString));
+
+        //set prune date TextView
+        TextView pruneDate = (TextView)findViewById(R.id.species_prune_date);
+        String pruneDateString = "<b>Prune Date:</b><br />"+garden.getPruneDate(speciesName);
+        pruneDate.setText(Html.fromHtml(pruneDateString));
+
+        //set size TextView
+        TextView size = (TextView)findViewById(R.id.species_size);
+        String sizeString = "<b>Size:</b><br />"  + garden.getSize(speciesName);
+        size.setText(Html.fromHtml(sizeString));
+
+        //set color TextView
+        TextView color = (TextView)findViewById(R.id.species_color);
+
+        String colorString = "<b>Color:</b><br />"
```

Comment:
I still think the user should see a circle with the selected color. We can accomplish this without any drawing. 
I'm thinking the unicode blacl circle symbol:    ●	9679	25CF 
We could bump the font size a little to make it more visible. 

Edit:
After speaking with you, it seems this causes aligment tests to fail.